components/_expanding-components/accordion-checkbox-dark.hbs

```diff
@@ -6,46 +6,45 @@
                 </div>
             </div>
             <div class="col-12 col-md-8">
```

Comment:
Why do we have cols in the middle of the component?

Reply:
Its how the accordion has been setup, this is something you and Pawel have both approved